lib/haml_lint/linter/rubocop.rb

```diff
@@ -117,7 +117,7 @@ def transfer_corrections(coordinator, new_ruby_code)
         # so the lints will be recorded then.
         @lints = []
 
-        msg = "Corrections couldn't be transfered: #{e.message} - Consider linting the file " \
+        msg = "Corrections couldn't be transferred: #{e.message} - Consider linting the file " \
```

Comment:
Note 1: this may be considered a breaking change
Note 2: as far as I understand, this is the only correct version in both British English and American English